openapi/components/schemas/Common/CommonBillingPortal.yaml

```
@@ -4,94 +4,108 @@ required:
   - slug
 properties:
   id:
-    description: The billing portal identifier string.
+    description: Billing portal identifier string.
     readOnly: true
     allOf:
       - $ref: ../ResourceId.yaml
   slug:
-    description: >-
-      The url path segment that will be appended to the billing portal
-      url (e.g. `https://rebilly.com/billing-portal/{slug}`) to make it human-readable and simple.
+    description: |-
+      Path segment that is appended to the billing portal URL to help
+      make it human-readable. Example: `https://company.com/billing-portal/{slug}`.
```

```suggestion
      make it human-readable. Example: `https://example.com/billing-portal/{slug}`.
```
We should not use a real URL in the example unless we have permission.